bin/oneoff/back_populate_user_geos

```diff
@@ -5,8 +5,25 @@
 
 require_relative '../../dashboard/config/environment'
 
-User.with_deleted.where.not(current_sign_in_ip: nil).find_each do |user|
-  next unless UserGeo.find_by_user_id(user.id).nil?
+if ARGV.length != 1
+  puts 'Usage: ./bin/oneoff/back_populate_user_geos start_index'
```

Comment:
Since there is a chance this will impact production and need to be cancelled, I'm wanting to allow for the possibility that the process gets killed. Restarting it with a `start_index` will prevent needless queries to confirm existing `UserGeo`s.